plugins/field-date/package.json

```diff
@@ -5,7 +5,9 @@
     "scripts": {
         "build": "gulp build",
         "clean": "rm -rf $npm_package_directories_dist",
-        "prepublishOnly": "npm run build"
+        "prepublishOnly": "npm run build",
+        "test": "npm run build && npm run test:mocha",
+        "test:mocha": "mocha --ui tdd test/test.mocha.js"
```

Comment:
Should this be using the blockly-scripts?

Reply:
Oh yes.

Reply:
No, as blockly-scripts is very much catered for the standard build environment. Field date uses closure to compile not webpack so it's unique.